js/repeater-list.js

```diff
@@ -202,6 +254,11 @@
 							self.list_positionHeadings();
 						}
 					});
+					if (self.viewOptions.list_frozenColumns) {
+						helpers.container.on('scroll.fu.repeaterFrozen', function () {
```

Comment:
This event should be `scroll.fu.repeaterList`. Probably a symptom of moving code from the repeater-frozen file into the repeater-list plugin